subprojects/docs/src/docs/userguide/artifactDependenciesTutorial.xml

```diff
@@ -71,44 +71,61 @@
 
     <section id="configurations">
         <title>Dependency configurations</title>
-        <para>In Gradle dependencies are grouped into <firstterm>configurations</firstterm>. A configuration is simply a named set of dependencies.
-            We will refer to them as <firstterm>dependency configurations</firstterm>. You can use them to declare the external dependencies of your project.
-            As we will see later, they are also used to declare the publications of your project.</para>
-        <para>
-            The Java plugin defines a number of standard configurations. These configurations represent the classpaths that the
-            Java plugin uses. Some are listed below, and you can find more details in <xref linkend="tab:configurations"/>.
+        <para>A Configuration is a named set of dependencies and artifacts. There are three main purposes for a Configuration:
+            <variablelist>
+                <varlistentry>
+                    <term>Declaring Dependencies</term>
+                    <listitem>
+                        <para>The plugin uses configurations to make it easy for build authors to declare what other subprojects or external artifacts are needed for various purposes
+                            during the execution of tasks defined by the plugin.</para>
+                    </listitem>
+                </varlistentry>
+                <varlistentry>
+                    <term>Resolving Dependencies</term>
+                    <listitem>
+                        <para>The plugin uses configurations to find (and possibly download) the files it needs from outside the project to perform the work of its tasks,
```

Comment:
I find this sentence hard to follow, especially the "coordinate and reason" part. The essence should be "These are used as task inputs".